frontend/workflows/projectSelector/src/project-selector.tsx

```diff
@@ -9,18 +9,32 @@ import LayersOutlinedIcon from "@material-ui/icons/LayersOutlined";
 import _ from "lodash";
 
 import { useDashUpdater } from "./dash-hooks";
-import { deriveStateData, DispatchContext, StateContext } from "./helpers";
+import {
+  deriveStateData,
+  DispatchContext,
+  exclusiveProjectDependencies,
+  StateContext,
+} from "./helpers";
 import ProjectGroup from "./project-group";
 import selectorReducer from "./selector-reducer";
-import { storeState } from "./storage";
+import { loadStoredState, storeState } from "./storage";
 import type { Action, DashState, State } from "./types";
 import { Group } from "./types";
 
+export interface ProjectSelectorError {
+  errors: IGoogle.rpc.IStatus[];
```

Comment:
Are these partial errors coming back from the server? If so it might be good to denote that in some way

Reply:
Yeah, they're from the partialFailures response. Ill add a docstring around it, we can rename it though if we think that'll be better